wix/Build.OpenJDK_generic.cmd

```diff
@@ -150,7 +146,7 @@ FOR %%A IN (%ARCH%) DO (
 					IF NOT EXIST "!REPRO_DIR!" (
 						ECHO Fourth !REPRO_DIR! not exists
 						REM try folder for JDK-Latest defined in CreateSourceFolder.AdoptOpenJDK.ps1
-						SET REPRO_DIR=.\SourceDir\!PRODUCT_SKU!-Latest\!PACKAGE_TYPE!\!FOLDER_PLATFORM!\!ADOPT_NAME!
+						SET REPRO_DIR=.\SourceDir\!PRODUCT_SKU!-Latest\!PACKAGE_TYPE!\!FOLDER_PLATFORM!\!PRODUCT_SHORT_VERSION!!JRE!
```

Comment:
I having call it ADOPT_NAME because it was not common to see jdk-PRODUCT_SHORT_VERSION-jre
jdk-..-jre ... this is strange naming (for JRE8)
but ok.. this does'nt change the folder path

Reply:
As far as I can tell, we never tried this folder location for JDK8 (before or after this change).

So you're right, this doesn't change things.